charts/authentik/templates/prometheusrule.yaml

```diff
@@ -127,30 +127,29 @@ spec:
         - alert: NoWorkersConnected
           labels:
             severity: critical
-          expr: max without (pid) (authentik_admin_workers) < 1
+          expr: max by (pod) (authentik_admin_workers{namespace="{{ $.Release.Namespace }}", service="{{ include "authentik.names.fullname" $ }}-metrics"}) < 1
```

Comment:
The service is authentik-server-metrics, not just authentik-metrics, or `{{ include "authentik.server.fullname" . }}-metrics`

Reply:
Probably not